Gems/Atom/Feature/Common/Assets/Materials/Types/StandardPBR_ForwardPass.azsl

```diff
@@ -43,320 +27,12 @@ COMMON_OPTIONS_EMISSIVE()
-        // m_specularColor render target 1, and the blend mode is (dest * source1color) + (source * 1.0).
-        //
-        // This means that m_specularColor.rgb (source 1) is multiplied against the destination, then
-        // m_diffuseColor.rgb (source) is added to that, and the final result is stored in render target 0.
-
-        lightingOutput.m_diffuseColor.rgb *= lightingOutput.m_diffuseColor.w; // pre-multiply diffuse
-
-        // Add specular. m_opacityAffectsSpecularFactor controls how much the alpha masks out specular contribution.
-        float3 specular = lightingOutput.m_specularColor.rgb;
-        specular = lerp(specular, specular * lightingOutput.m_diffuseColor.w, MaterialSrg::m_opacityAffectsSpecularFactor);
-        lightingOutput.m_diffuseColor.rgb += specular;
-
-        lightingOutput.m_specularColor.rgb = baseColor * (1.0 - alpha); 
-    }
-    else
-    {
-        lightingOutput.m_diffuseColor.w = -1; // Disable subsurface scattering
-    }
-
-    return lightingOutput;
-}
-
-ForwardPassOutputWithDepth StandardPbr_ForwardPassPS(VSOutput IN, bool isFrontFace : SV_IsFrontFace)
-{
-    ForwardPassOutputWithDepth OUT;
-    float depth;
-
-    PbrLightingOutput lightingOutput = ForwardPassPS_Common(IN, isFrontFace, depth);
-
-#ifdef UNIFIED_FORWARD_OUTPUT
-    OUT.m_color.rgb = lightingOutput.m_diffuseColor.rgb + lightingOutput.m_specularColor.rgb;
-    OUT.m_color.a = lightingOutput.m_diffuseColor.a;
-    OUT.m_depth = depth;
-#else
-    OUT.m_diffuseColor = lightingOutput.m_diffuseColor;
-    OUT.m_specularColor = lightingOutput.m_specularColor;
-    OUT.m_specularF0 = lightingOutput.m_specularF0;
-    OUT.m_albedo = lightingOutput.m_albedo;
-    OUT.m_normal = lightingOutput.m_normal;
-    OUT.m_depth = depth;
-#endif
-    return OUT;
-}
-
-[earlydepthstencil]
-ForwardPassOutput StandardPbr_ForwardPassPS_EDS(VSOutput IN, bool isFrontFace : SV_IsFrontFace)
-{
-    ForwardPassOutput OUT;
-    float depth;
-
-    PbrLightingOutput lightingOutput = ForwardPassPS_Common(IN, isFrontFace, depth);
+#include "MaterialFunctions/EvaluateStandardSurface.azsli"
+#include "MaterialFunctions/EvaluateTangentFrame.azsli"
+#include "MaterialFunctions/ParallaxDepth.azsli"
+#include "MaterialFunctions/StandardGetNormalToWorld.azsli"
+#include "MaterialFunctions/StandardGetObjectToWorld.azsli"
+#include "MaterialFunctions/StandardGetAlphaAndClip.azsli"
+#include "MaterialFunctions/StandardTransformUvs.azsli"
 
-#ifdef UNIFIED_FORWARD_OUTPUT
-    OUT.m_color.rgb = lightingOutput.m_diffuseColor.rgb + lightingOutput.m_specularColor.rgb;
-    OUT.m_color.a = lightingOutput.m_diffuseColor.a;
-#else
-    OUT.m_diffuseColor = lightingOutput.m_diffuseColor;
-    OUT.m_specularColor = lightingOutput.m_specularColor;
-    OUT.m_specularF0 = lightingOutput.m_specularF0;
-    OUT.m_albedo = lightingOutput.m_albedo;
-    OUT.m_normal = lightingOutput.m_normal;
-#endif
-    return OUT;
-}
+#include "StandardSurface_ForwardPass.azsli"
```

Comment:
Not clear from this PR why StandardSurface_ForwardPass needs to exist outside StandardPBR_ForwardPass.azsl when it's only included there. Comments around these cases where one file/function is only included/used once would help clarify parts of this refactor and the intention going forward.

Reply:
https://github.com/o3de/o3de/pull/7437#discussion_r801076507